.github/workflows/ci.yaml

```diff
@@ -214,3 +214,5 @@ jobs:
           else
             echo "Publish step skipped."
           fi
+#      - name: Setup upterm session
+#        uses: lhotari/action-upterm@v1
```

Comment:
You should configure your IDE to auto add new line at the end of file

Reply:
yep